.reuse/dep5

```diff
@@ -5,3 +5,7 @@ Source: https://github.com/edgehog-device-manager/edgehog-device-runtime
 Files: CHANGELOG.md
 Copyright: 2022 SECO Mind Srl
 License: CC0-1.0
+
+Files: edgehog-forwarder/interfaces/*
+Copyright: 2022 SECO Mind Srl
+License: CC0-1.0
```

Comment:
Remove these lines, the interfaces should be added  into the [edgehog-astarte-interfaces repo](https://github.com/edgehog-device-manager/edgehog-astarte-interfaces).
```suggestion
```

Reply:
Done